### DISCUSSION TITLE
accidentally giving extension A third-party OAuth application with gist and read:user scopes was recently authorized to access your account.

### DISCUSSION THREAD

**Discussion by jamesablanco** [community/community#31872] [New to GitHub] [answered]
I am new to VS Code and coding, I was installing extensions and I wasn't paying attention to a home screen feature and wanted to sync my GitHub but didn't realize I was giving an extension permission to my authorization 

"
accidentally giving extension A third-party OAuth application with gist and read:user scopes was recently authorized to access your account."

If I uninstall the extension is everything alright with my account? I looked under settings and did not see a person or group getting access.

Sorry for being new to this 

Thanks, 
J

**Comment by airtower-luna** [comment, marked as answer]
What uninstalling the extension does or doesn't do depends on the extension itself, so it's impossible to answer in general. It *could* revoke its own access, but you'd have to check the code to be sure.

If you want to be sure check under Settings > Applications, and there the "Authorized OAuth Apps" tab. It shows you a list of authorized applications, and you can revoke any you don't want.

**Comment by jamesablanco** [comment]
Ahhh Thank you so much, I figured out how to revoke it now! 

I appreciate this community so much! 

Cheers, 
J

**Reply by airtower-luna** [reply]
Excellent! Marking the answer would be appreciated. :slightly_smiling_face:

**Reply by jamesablanco** [reply]
Done Thank you! learning this platform better and better

**Comment by marciakaren-smithabramsalmasi** [comment]
I received this message...[GitHub] A first-party GitHub OAuth application has been added to your account.
I was working on trying to sync my git hub account with visual studio code.  I did not want this to happen.  How to remove it?
